crates/primitives/src/explorers/blockscout.rs

```diff
@@ -31,10 +31,10 @@ impl BlockScout {
         })
     }
 
-    pub fn new_hyper_evm() -> Box<Self> {
+    pub fn new_hyperliquid() -> Box<Self> {
         Box::new(Self {
             meta: Metadata {
-                name: "Hyperliquid EVM Explorer",
+                name: "BlockScout",
```

Comment:
```suggestion
                name: "BlockScout",
```

can you add to static vars? to reuse

Reply:
done